### PR TITLE
docs: Change in setting up a debug console

### PR DESCRIPTION
**Pull request by Amulyam24**
The debug console can be set up with the help of both initrd or rootfs image.

Signed-off-by: Amulya Meka <amulmek1@in.ibm.com>

### CODE REVIEW

File: Developer-Guide.md [hunk 1]
```diff
@@ -451,7 +451,7 @@ packages in the rootfs, which would increase the size of the image used to
 boot the virtual machine.
 
 If you want to login to a virtual machine that hosts your containers, complete
-the following steps, which assume the use of a rootfs image.
+the following steps(using rootfs or initrd image).
```
Comment (grahamwhaley) [inline Developer-Guide.md]: You need a space between `steps` and the `(`

File: Developer-Guide.md [hunk 2]
```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
 ### Build the debug image
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs and for initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
```
Comment (grahamwhaley) [inline Developer-Guide.md]: I'd reword this just slightly different to make it flow better:
> section when using rootfs, or when using initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.

File: Developer-Guide.md [hunk 3]
```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
...
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: The image has to be changed from `kata-containers.img` to `kata-containers-initrd.img` while using initrd.
```
Comment (grahamwhaley) [inline Developer-Guide.md]: Minor rewording maybe, to avoid any confusion about renaming the image:

> **Note**: When using an initrd image, use the image name `kata-containers-initrd.img` below, in place of the rootfs image name `kata-containers.img`.

**Comment by Amulyam24** [comment]
Thank you @grahamwhaley. Will make the required changes.

**Comment by grahamwhaley** [comment]
np @Amulyam24 - I also meant to say, if you could merge your commits when they are related, so we end up with the minimum number, that'd also be great. So, a combination of `git rebase -i` to fold them together and `git push -f` to push the changes back up to github. There are some docs on the process over at https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#updating-your-pr-based-on-review-comments

**Comment by Amulyam24** [comment]
Yes. I'll look into it and merge the commits.

File: Developer-Guide.md [hunk 4]
```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
...
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs, or for initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: When using an initrd image, use the image name `kata-containers.img` below, in place of the rootfs image name `kata-containers.img`.
```
Comment (chavafg) [inline Developer-Guide.md]: nit: the first `kata-containers.img` should be `kata-containers-initrd.img`

Comment (Amulyam24) [reply Developer-Guide.md]: Thank you @chavafg. Will change it.

Comment (chavafg) [reply Developer-Guide.md]: thanks @Amulyam24 but I still see this line needs to be changed, you have:

>**Note**: When using an initrd image, use the image name `kata-containers.img` below, in place of the rootfs image name `kata-containers-initrd.img`.

And should be:

> **Note**: When using an initrd image, use the image name `kata-containers-initrd.img` below, in place of the rootfs image name `kata-containers.img`.

am I correct @grahamwhaley ? Or I'm getting confused because of the wording?  😄

Comment (grahamwhaley) [reply Developer-Guide.md]: I think you are correct @chavafg . I would probably write it as:

> **Note:** When using an initrd image, replace the below rootfs image name `kata-containers.img` with the initrd image name `kata-containers-initrd.img`.

Comment (Amulyam24) [reply Developer-Guide.md]: Yes. That has to be changed. Thanks @chavafg

File: Developer-Guide.md [hunk 5]
```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
...
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs, or when using initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: When using an initrd image, replace the below rootfs image name `kata-containers.img` with the initrd image name `kata-containers-initrd.img`.
```
Comment (jodh-intel) [inline Developer-Guide.md]: This needs a leading `>` so that it render as:

> **Note**: When using an ...

See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes.

Comment (Amulyam24) [reply Developer-Guide.md]: Thanks @jodh-intel . I've made the required change.
